redistools/conn.py

```diff
@@ -2,6 +2,12 @@
 from redis import StrictRedis
 from redis.sentinel import Sentinel
 
+TRACE = os.getenv('TRACE_REDIS', 'false') == "true"
+
+if TRACE:
+    import inspect
+    logger_function = print
```

Comment:
Can we throw to `logger.debug` instead?

Reply:
there's no logger setup for redistools rn or I would have done so! maybe a good next step.

Reply:
wanna throw it in? pretty quick right?

Reply:
sure, only when TRACE though.

Reply:
cool!